test/cli/invalid-project/invalid-project.vcxproj

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
hmm.. I have two alternative ideas:
 * generate vcxproj file from the test in a temporary folder
 * copy helloworld.vcxproj to a temporary folder and try to check that.